Formula/r.rb

```diff
@@ -0,0 +1,102 @@
+class R < Formula
+  desc "Software environment for statistical computing"
+  homepage "https://www.r-project.org/"
+  url "https://cran.rstudio.com/src/base/R-3/R-3.4.1.tar.gz"
+  sha256 "02b1135d15ea969a3582caeb95594a05e830a6debcdb5b85ed2d5836a6a3fc78"
+
+  bottle do
+    sha256 "d0254993416c177d7fa49b9cde95eb8bd262e3a801408b21951cc0f7755e0a0e" => :sierra
+    sha256 "2098376a2d552573a1b0e2ff29c076b05a0161ec276260b5b76a80e87d5cd6c1" => :el_capitan
+    sha256 "be31e78c3df77a46e91500b4809cb7f89bceacabc0c38d1bc3e56beab31bff6e" => :yosemite
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "texinfo" => :build
+  depends_on "gettext"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "pcre"
+  depends_on "readline"
+  depends_on "xz"
+  depends_on :fortran
+  depends_on "homebrew/science/openblas" => :optional
+  depends_on "pango" => :optional
+  depends_on :java => :optional
+
+  def install
+    # Fix dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
+    if MacOS.version == "10.11" && MacOS::Xcode.installed? &&
+       MacOS::Xcode.version >= "8.0"
+      ENV["ac_cv_have_decl_clock_gettime"] = "no"
+    end
+
+    args = [
+      "--prefix=#{prefix}",
+      "--enable-memory-profiling",
+      "--without-cairo",
+      "--without-x",
+      "--with-aqua",
+      "SED=/usr/bin/sed", # don't remember Homebrew's sed shim
+    ]
+
+    if build.with? "openblas"
+      args << "--with-blas=-L#{Formula["openblas"].opt_lib} -lopenblas"
+      args << "--with-lapack"
```

Comment:
looks like we're adding `--with-lapack` in both the `if` and the `else` so I think it belongs in the top array.